website/integrations/services/semaphore/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate with Semaphore UI
+sidebar_label: semaphore
+---
+
+# Semaphore UI
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Semaphore UI
+
+> Semaphore UI is a modern web interface for managing popular DevOps tools.
+> -- https://semaphoreui.com/
+>
+> This guide explains how to set up Semaphore UI to use authentik as the OAuth provider for the login to the WebGUI.
+
+## Preparation
+
+The following placeholders will be used:
+
+- `semaphore.company` is the FQDN of the engomo install.
+- `authentik.company` is the FQDN of the authentik install.
+- `ak.cert` is the self-signed certificate that will be used for the service provider.
+
+## Semaphore UI configuration
```

Comment:
Did you mean to title this "authentik configuration"?

Reply:
HI @nicedevil007 help me understand this one.. don't we want line 24 to be "authentik configuration", because this section is about creating the app and provider in authentik. Then line 45 is the Semaphore section, no?